### PR TITLE
osx/codesign.md: make placeholders auto convertible to CLIP ones

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

## Note

`file` suffix is required to make placeholder convertible.

## Result

Conversion result thanks to fixes:

```md
# codesign

> Create and manipulate code signatures for macOS
> More information: https://www.unix.com/man-page/osx/1/codesign/

- Sign an application with a certificate:

`codesign --sign "{string some description: My Company Name}" {file application file with mandatory .app extension}`

- Verify the certificate of an application:

`codesign --verify {file application file with mandatory .app extension}`

```

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
I am not really sure about volume_file placeholder here. Let's see what other think.

File: pages/osx/asr.md [hunk 1]
```diff
@@ -6,16 +6,16 @@
 
 - Restore a disk image to a target volume:
 
-`sudo asr restore --source {{image_name}}.dmg --target {{path/to/volume}}`
+`sudo asr restore --source {{image_filename.dmg}} --target {{path/to/volume_file}}`
```
Comment (blueskyson) [inline pages/osx/asr.md:9]:
```suggestion
`sudo asr restore --source {{image_file.dmg}} --target {{path/to/volume_file}}`
```

File: pages/osx/asr.md [hunk 2]
```diff
@@ -6,16 +6,16 @@
 
 - Restore a disk image to a target volume:
 
-`sudo asr restore --source {{image_name}}.dmg --target {{path/to/volume}}`
+`sudo asr restore --source {{image_file.dmg}} --target {{path/to/volume_file}}`
 
 - Erase the target volume before restoring:
 
-`sudo asr restore --source {{image_name}}.dmg --target {{path/to/volume}} --erase`
+`sudo asr restore --source {{image_filename.dmg}} --target {{path/to/volume_file}} --erase`
```
Comment (blueskyson) [inline pages/osx/asr.md:13]:
```suggestion
`sudo asr restore --source {{image_file.dmg}} --target {{path/to/volume_file}} --erase`
```

File: pages/osx/asr.md [hunk 3]
```diff
@@ -6,16 +6,16 @@
...
 
 - Skip verification after restoring:
 
-`sudo asr restore --source {{image_name}}.dmg --target {{path/to/volume}} --noverify`
+`sudo asr restore --source {{image_filename.dmg}} --target {{path/to/volume_file}} --noverify`
```
Comment (blueskyson) [inline pages/osx/asr.md:17]:
```suggestion
`sudo asr restore --source {{image_file.dmg}} --target {{path/to/volume_file}} --noverify`
```

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**